Source/WebCore/platform/MediaStrategy.cpp

```diff
@@ -51,6 +51,11 @@ void MediaStrategy::resetMediaEngines()
     m_mockMediaSourceEnabled = false;
 }
 
+bool MediaStrategy::hasThreadSafeMediaSupport() const
```

Comment:
```suggestion
bool MediaStrategy::hasThreadSafeMediaSourceSupport() const
```

Reply:
It's the MediaPlayerPrivateRemote itself that needs to be thread-safe, not just MSE code. but seeing indeed it's only used in combination with MSE, I'll consider that name